schedule/yast/textmode/textmode.yaml

```diff
@@ -2,41 +2,23 @@
 name: textmode
```

Comment:
Please move the yaml file to schedule/yam/test_cases/  as a single yaml file.

Reply:
I think it's fixed